src/test/groovy/graphql/validation/SpecValidation573Test.groovy

```diff
@@ -0,0 +1,72 @@
+package graphql.validation
+/**
+ * validation examples used in the spec in given section
+ * http://facebook.github.io/graphql/#sec-Validation
+ *
+ */
+class SpecValidation573Test extends SpecValidationBase {
+
+    def '5.7.3 Variables Are Input Types - type mismatch (must be non-null)'() {
+        def query = """
```

Comment:
If you want to use $ in a groovy multi line string, then just use ''' quotes instead.  No need for quoting the \$

Reply:
oh neat I didn't know you could do that! Thanks for the merge by the way.